src/pages/nowplaying/index.jsx

```diff
@@ -0,0 +1,55 @@
+import hero from "../../assets/imgs/Hero - Image.png" ;
+import heart from "../../assets/imgs/like-heart.png";
+import line1 from "../../assets/imgs/Line1.png" 
+import line2 from "../../assets/imgs/Line2.png"
+import circle from "../../assets/imgs/circle.png"
+import repeate from "../../assets/imgs/repeate.png"
+import previous from "../../assets/imgs/previous.png" 
+import buttonnowplaying from "../../assets/imgs/buttonnowplaying.png"
+import next from "../../assets/imgs/next.png"
+import Shuffle from "../../assets/imgs/Shuffle.png"
+
+import './style.css';
+
+function NowPlayimg(){
```

Comment:
Não seria seria com N?
```suggestion
function NowPlaying(){
```